examples/django_chat/sdjango/sd_manager.py

```diff
@@ -0,0 +1,22 @@
+from socketio.base_manager import BaseManager
+
+
+class SdManager(BaseManager):
```

Comment:
The `BaseManager` class is not supposed to be used from the outside. Besides, this subclass that you created doesn't do anything, so why do you have it?